ts/src/binance.ts

```diff
@@ -9091,4 +9093,211 @@ export default class binance extends Exchange {
             'info': interest,
         };
     }
+
+    async fetchMyLiquidations (symbol: string, since: Int = undefined, limit: Int = undefined, params = {}) {
```

Comment:
also, symbol is not mandatory here, so we should update the method's signature 
![image](https://github.com/ccxt/ccxt/assets/43336371/63fcb1f4-daad-4213-9307-90e818ebf02b)